src/Cli/dotnet/commands/dotnet-new/NewCommandShim.cs

```diff
@@ -66,6 +66,7 @@ private static ITemplateEngineHost CreateHost(bool disableSdkTemplates)
             if (!disableSdkTemplates)
             {
                 builtIns.Add(new KeyValuePair<Guid, Func<Type>>(BuiltInTemplatePackageProviderFactory.FactoryId, () => typeof(BuiltInTemplatePackageProviderFactory)));
+                builtIns.Add(new KeyValuePair<Guid, Func<Type>>(OptionalWorkloadProviderFactory.FactoryId, () => typeof(OptionalWorkloadProviderFactory)));
```

Comment:
wondering if we should have different command for disabling workloads :eyes:

Reply:
I was considering this... My reasoning for using same flag is... Its very unlikely user of this argument would not want to disable optional workloads, its more likely they would disable sdk, but not optional workloads, and then be surprised why something showed up...
Also adding new flag would require another round of changes on templating and bumping into sdk... I would prefer to avoid that...